tensorflow/core/framework/allocator_registry.cc

```diff
@@ -25,12 +27,27 @@ AllocatorRegistry* AllocatorRegistry::Global() {
   return global_allocator_registry;
 }
 
-void AllocatorRegistry::Register(const string& name, uint8_t priority,
+bool AllocatorRegistry::CheckForDuplicates(const string& name, int priority) {
+  for (std::vector<AllocatorRegistryEntry>::iterator it = allocators_.begin();
+       it != allocators_.end(); ++it) {
+    if (!name.compare(it->name) && it->priority == priority) return true;
```

Comment:
Change this to 

 if (!name.compare(it->name) && it->priority == priority) {
  return true;
}